templates/galaxy/config/container_resolvers_conf.yml.j2

```diff
@@ -0,0 +1,128 @@
+# Sample file for container resolvers. Resolvers that are potentially used by default
```

Comment:
```suggestion
# Configuration file for container resolvers. Resolvers that are potentially used by default
```

Just because the file is used in production now.